.github/actions/dbp-charts/verify-helm/helm_install.sh

```diff
@@ -11,13 +11,11 @@ clean_up () {
 }
```

Comment:
we are deleting the ingress release but not the acs one. IS there a reason for that?

Reply:
I got fooled by that too, double check the helm delete argument(s)

do you think for clarity it's better to have two different helm delete commands, one for each release?

Reply:
Ah ok, no it's OK now we know it :)